applications/nrf_desktop/src/modules/Kconfig.config_channel

```diff
@@ -88,6 +96,46 @@ config DESKTOP_CONFIG_CHANNEL_DFU_MCUBOOT_DIRECT_XIP
 	  the image with a higher version. The module does not mark the newly
 	  uploaded image as pending nor confirm it after a successful boot.
 
+config DESKTOP_CONFIG_CHANNEL_DFU_MCUBOOT_RAM_LOAD
+	bool "Adjust the DFU module to be compatible with the RAM load mode of MCUboot"
+	depends on USE_DT_CODE_PARTITION
+	depends on $(dt_nodelabel_exists,cpuapp_sram_mcuboot_ram_region)
```

Comment:
Consider using BUILD_ASSERT instead of dependency here (now you might end up with accidentally set `CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_MCUBOOT_UNSPECIFIED`)